shell/platform/darwin/ios/framework/Source/FlutterSpellCheckPlugin.mm

```diff
@@ -88,13 +102,12 @@ - (void)handleMethodCall:(FlutterMethodCall*)call result:(FlutterResult)result {
     }
   } while (nextSpellSuggestion != nil && nextOffset < text.length);
 
-  NSMutableArray* methodChannelResult = [[[NSMutableArray alloc] init] autorelease];
+  NSMutableArray* methodChannelResult = [[NSMutableArray alloc] init];
```

Comment:
Optional nit: it would be nice to make this an `initWithCapacity:` to avoid pointless re-allocations in the loop, since we know the length will be the same as `allSpellSuggestions`.

Reply:
Done.